processor/transform/reusable-content.js

```diff
@@ -0,0 +1,29 @@
+import { visit } from 'unist-util-visit';
+
+const type = 'reusable-content';
+
+const regexp = /^\s*<ReadMeReusableContent name="(?<name>.*)" \/>\s*$/;
+
+const reusableContentTransformer = function () {
+  const reusableContent = this.data('reusableContent');
```

Comment:
What does this do?

Reply:
Oh, just passing in the map of reusable content blocks, so I don't have to pass it directly to the transformer.